logstash-core/lib/logstash/api/lib/app/commands/stats/events_command.rb

```diff
@@ -8,9 +8,9 @@ def run
     #need to be tailored to the right metrics for this command.
     stats = service.get(:events_stats)
     {
-      :in => stats[:base][:events_in].value,
-      :out => 0,
-      :dropped => stats[:base][:events_filtered].value
+      :in => stats[:stats][:events][:in].value,
+      :out => stats[:stats][:events][:out].value,
+      :dropped => stats[:stats][:events][:filtered].value
```

Comment:
Lets agree on a definition for dropped here.

For me `dropped` events are events that we have removed from the pipeline, either a plugin decided to remove the events from the pipeline or a event became poisonous, or an event raised an exception.